tip-wlan/charts/wlan-cloud-graphql-gw/templates/ingress.yaml

```diff
@@ -32,6 +32,12 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+        {{- if $.Values.ingress.alb_https_redirect }}
+          - path: /*
```

Comment:
Why do we need a feature flag "alb_https_redirect". Can we just specify these data inside the `.paths` variable? This way, we won't add AWS specific parameters to the Helm charts.

Reply:
that would be great option but would require to refactor all value files too (+ testing), adding just extra var helps making this change as small as possible...